README.md

```diff
@@ -548,7 +548,7 @@ public struct AudioSpeechQuery: Codable, Equatable {
 **Response:**
 
 ```swift
-/// Audio data for one of the following formats :`mp3`, `opus`, `aac`, `flac`
+/// Audio data for one of the following formats :`mp3`, `opus`, `aac`, `flac`, `pcm`
```

Comment:
Interestingly the documentation also mentions `wav`, so I'm wondering if we should add that as well to this PR or a separate PR. 

![image](https://github.com/MacPaw/OpenAI/assets/1451896/e1a6c38f-2536-4392-9eca-82575988d29f)